RELEASES.md

```diff
@@ -1,5 +1,15 @@
 --------------------------------------------------------------------------------
 
+## 18.0.2
+
+### Fixed
+
+* Fix an egraph rule bug that was permitting an incorrect `ireduce` rewrite to
+  unary and binary operations, leading to miscompilations.
+  [#8005](https://github.com/bytecodealliance/wasmtime/pull/8005)
+
+--------------------------------------------------------------------------------
+
 ## 18.0.1
 
 Released 2024-02-20.
```

Comment:
For when 18.0.2 and 17.0.2 are released mind adding these date-related "Released" entries in the notes?

(you should be able to push directly to the PR created from CI when doing that)